index.html

```diff
@@ -8,6 +8,21 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <center>
+        <section id="header">
+            <img class ="image_geometry_1" src="images/image_geometry_1.png"/>
+           
+            <img class ="ic_menu" src="images/ic_menu.png"/>
+
+        </section>
```

Comment:
- [ ] I think it will be better if you wrap the content of your toolbar in a `header `tag instead of a` section` tag because the `<header>` element represents a container for introductory content or a set of navigational links 👍🏽 

e.g `<header id="header">`